src/api/app/views/webui2/webui/request/show.html.haml

```diff
@@ -35,13 +35,31 @@
 .row
   .col-md-8
     .card.mb-3
-      .card-header.d-flex.justify-content-between
-        %h5
-          Comments for request #{@number}
-          %span.badge.badge-primary
-            = @comments.size
-      .card-body#comments
-        = render partial: 'webui2/webui/comment/show', locals: { commentable: @bs_request }
+      .bg-light
+        %ul.nav.nav-tabs{ role: 'tablist' }
+          %li.nav-item
+            = link_to('#comments', class: 'nav-link text-nowrap active', data: { toggle: 'tab' }, role: 'tab') do
+              Comments for request #{@number}
+              %span.badge.badge-primary
+                = @comments.size
+          - if @superseding
+            - @superseding.includes(:comments).each do |superseded|
+              - if superseded.comments.any?
+                %li.nav-item
+                  = link_to("#superseded-#{superseded.number}", title: "This request was superseded by request #{superseded.superseded_by}",
+                    class: 'nav-link text-nowrap', data: { toggle: 'tab' }, role: 'tab') do
+                    Comments for request #{superseded.number}
+                    %span.badge.badge-primary
+                      = superseded.comments.size
+      .card-body
+        .tab-content
+          .tab-pane.fade.show.active#comments
+            = render partial: 'webui2/webui/comment/show', locals: { commentable: @bs_request }
+          - if @superseding
```

Comment:
[nitpick] This can leave if we just get rid of the `unless` in `result['superseding'] = superseding unless superseding.empty?` `webui_infos` again....